impr: RULE- 197 update ruleSettings.md to improve it (#145)
cloudconformity/documentation-api: RuleSettings.md

  //...
}
```

###### multiple-vpc-gateway-mappings
* **Usage:** Used when one or more VPC gateway mappings are required.
* **UI:** List of VPC Id and Gateway Id mapping

_Example:_
```json5
{
  "id": "VPC-013",
  //...
  "extraSettings": [
    {
      "type": "multiple-vpc-gateway-mappings",
      "name": "SpecificVPCToSpecificGatewayMapping",
      "mappings": [
        {
          "values": [
            {
              "type": "single-string-value",
              "name": "vpcId",
              "value": "vpc-001"
            },
            {
              "type": "multiple-string-values",
              "name": "gatewayIds",
              "values": [
                {
                  "value": "nat-001"
                },
                {
                  "value": "nat-002"
                 }
              ]
            }
          ]
        }
            //...
      ]
    }
  ],
  //...
}
```

## Get Rule Settings By Id

A GET request to this endpoint allows you to get rule settings based on the rule Id.

##### Endpoints:

`GET /v1/accounts/{accountId}/settings/rules/{ruleId}`

> ##### Paramters

Reviewer: ⚠️ 
```suggestion
##### Parameters
```

> - `accountId`: Cloud Conformity ID of the account. Provide to get only settings set for the specified account.

Reviewer: 🎨 
```suggestion
- `accountId`: Cloud Conformity account ID. Provide to get only rule settings for the specified account.
```

Reviewer: I think this is a bit clearer. What do you think?

> - `ruleId`: Cloud Conformity Rule ID. Provide to get only settings set for the specified rule.

Reviewer: ```suggestion
- `ruleId`: Cloud Conformity Rule ID. Provide to get only rule settings for the specified rule.
```

                      {
                        "type": "single-string-value",
                        "name": "vpcId",
                        "value": "vpc-002"
                      },
                      {
                        "type": "multiple-string-values",
                        "name": "gatewayIds",
                        "values": [
                          {
                            "value": "nat-002"
                          }
                        ]
                      }
                    ]
                  }
                ]
              }
            ],
            "provider": "aws",
            "id": "VPC-013",
            "enabled": true,
            "exceptions": {
              "resources": null,
              "tags": null
            }
          }
        ],
        "access": {}
      },
      "access": null,
      "cloud-type": "aws"
    },
    "relationships": {
      "organisation": {
        "data": {
          "type": "organisations",
          "id": "organisationId"
        }
      }
    }
  }
}

```

## Update Rule Settings

A PATCH request to this endpoint allows you to update rule settings.

##### Endpoints:

> `PATCH accounts/{accountId}/settings/rules`

Reviewer: ⚠️ 
does this not use the `/v1/` route?

```suggestion
`PATCH /v1/accounts/{accountId}/settings/rules`
```

Author: To keep it consistent like other api endpoints, we don't include the version.

##### Parameters

> - `data`: A JSON object containing JSONAPI compliant data object with following properties

Reviewer: ⚠️  could we add `accountId` here, like we do above? [Looking at other documentation pages](https://github.com/cloudconformity/documentation-api/blob/RULE-197/Groups.md#update-group) it doesn't seem to matter that we mix query params and the requests body under the same 'Parameters' header

Author: accountId parameter info is added.

    "attributes": {
      "settings": {
        "rules": [
          {
            "riskLevel": "MEDIUM",
            "id": "S3-019",
            "extraSettings": [
              {
                "name": "s3_buckets",
                "label": "S3 Buckets",
                "type": "multiple-string-values",
                "values": [
                  {
                    "value": "test-bucket"
                  },
                  {
                    "value": "test-bucket-2"
                  }
                ]
              }
            ],
            "provider": "aws",
            "enabled": true,
            "exceptions": {
              "resources": null,
              "tags": null
            }
          }
        ],
        "access": {}
      },
      "access": null,
      "cloud-type": "aws"
    },
    "relationships": {
      "organisation": {
        "data": {
          "type": "organisations",
          "id": "organisationId"
        }
      }
    }
  }
}
```

## Update Rule Settings By Id

A PATCH request to this endpoint allows you to update a specific rule's settings.

##### Endpoints:

`PATCH accounts/{accountId}/settings/rules/{ruleId}`

##### Parameters

> - `data`: A JSON object containing JSONAPI compliant data object with following properties

Reviewer: ⚠️ for consistency can we add the query params here as well?

Author: query params info is added.
